freqtrade/exchange/bitget.py

```diff
@@ -50,41 +49,6 @@ class Bitget(Exchange):
         (TradingMode.FUTURES, MarginMode.ISOLATED)
     ]
 
-    @property
-    def _ccxt_config(self) -> dict:
-        config = {}
-        if self.trading_mode == TradingMode.SPOT:
-            config.update({"options": {"defaultType": "spot"}})
-        elif self.trading_mode == TradingMode.FUTURES:
-            config.update({"options": {"defaultType": "swap"}})
-        config.update(super()._ccxt_config)
-        return config
-
-    @retrier
-    def additional_exchange_init(self) -> None:
-        try:
-            if not self._config["dry_run"]:
-                if self.trading_mode == TradingMode.FUTURES:
-                    # Set position mode to one-way (hedged = False)
-                    # self._api.set_position_mode(False, None, {'productType': 'USDT-FUTURES'})
```

Comment:
I didn't necessarily mean this line - just the "defaultType" settings below.
this **might** be fine - though in reality, you'll probably want to query the position mode (check if it's set to one-way and fail to start otherwise).

Force-setting it can fail miserably if there's an open position already - either failing to switch it - or by force-closing all open positions (at least that's my experience with other exchanges).